modules/zone/zone.go

```diff
@@ -93,82 +56,86 @@ func (s *Lookup) DoZonefileLookup(record *dns.Token) (interface{}, zdns.Status,
 	lookup, _ := s.Factory.Subfactory.MakeLookup()
 	var nameserver string
```

Comment:
Why is `lookup` defined here? This isn't used until L103, and there are exit branches in the function between here and then.